src/transaction/atomic_replication_helper.cpp

```diff
@@ -245,12 +246,17 @@ namespace cublog
     PGBUF_CLEAR_WATCHER (&m_watcher);
   }
 
-  void
+  int
```

Comment:
revert the return to `void` and revert the rest to adapt; it is enough that we log a fatal error which will also crash the process